<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer_diag</class>
 <widget class="QDialog" name="Timer_diag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
     <item>
      <widget class="QSpinBox" name="spinBox_h">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #fffdd0;
color: #ff00ff;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <number>23</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_m">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #fffdd0;
color: #ff00ff;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_s">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeVerCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #fffdd0;
color:#ff00ff;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="Add_btn">
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
</string>
     </property>
     <property name="text">
      <string>Add Timer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
